analytics.go

```diff
@@ -17,7 +18,11 @@ import (
 )
 
 // Version of the client.
-const Version = "3.4.0"
+const Version = "4.0.0"
```

Comment:
How are going to document this breaking change? Don't we need something like a *Versions & Compatibility* section in the readme?

Reply:
We can add.